src/ibek/ioc_cmds/assets.py

```diff
@@ -45,7 +45,13 @@ def extract_assets(
     """
     extract and copy runtime assets from a completed developer stage container
     """
-    asset_matches = "bin|configure|db|dbd|include|lib|template|config|*.sh"
+
+    if GLOBALS.STATIC:
+        # static builds only need database files from support modules
+        asset_matches = "db|template|*.sh"
+    else:
+        # dynamically linked builds need binaries
+        asset_matches = "bin|configure|db|dbd|include|lib|template|config|*.sh"
```

Comment:
This all worked out nice if  i copy the contents of the IOC folder. so special handling for non native builds required (because we also don't need to copy the venv for native anymore)

Reply:
This also allowed me to pare asset matches down to "bin|db|lib"